### PR TITLE
feat(docs): update "new component" readme

### PR DESCRIPTION
**Pull request by dmitry-stepanenko**
## Related Issue(s)

n/a

## Description
After creating the schematic to generate new components, the process has been significantly simplified. It's now required to update the docs for creation of new components to be up-to-date. 

##### During Implementation

1. Visual Testing:

-   [n/a] visual misalignments/updates
-   [n/a] check Light/Dark/HCB/HCW themes
-   [n/a] RTL/LTR - proper rendering and labeling
-   [n/a] responsiveness(resize)
-   [n/a] Content Density (Cozy/Compact/(Condensed))
-   [n/a] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [n/a] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [n/a] Mouse vs. Keyboard support
-   [n/a] Text Truncation

2. API and functional correctness

-   [x] check for console logs (warnings, errors)
-   [n/a] API boundary values
-   [n/a] different combinations of components - free style
-   [n/a] change the API values during testing

3. Documentation and Example validations

-   [x] missing API documentation or it is not understandable
-   [x] poor examples
-   [x] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [n/a] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [n/a] Run npm run build-pack-library and test in external application
-   [x] update `README.md`
-   [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

File: NEW_COMPONENT.md [hunk 1]
```diff
@@ -1,231 +1,110 @@
...
+-   docs section with default example
 
-## Step2: Build module & component within libs directory
+Navigate to http://localhost:4200/fundamental-ngx#/core/poster-editor to start working with component.
 
-We're going to be building a component called 'Poster' that displays images. Generate a new 'poster' module in the `src/lib` directory.
+Component code will be located at `libs/core/src/lib/poster-editor`, and the example can be found at `apps/docs/src/app/core/component-docs/poster-editor/examples/default`.
 
-`ng generate module core/src/lib/poster`
+At this point point you can start coding: apply changes in the library component and test them in docs right away.
```
Comment (N1XUS) [inline NEW_COMPONENT.md:27]: duplicated 'point'

**Review by N1XUS — APPROVED** [review]

File: NEW_COMPONENT.md (hunk 1 above)
Comment (dmitry-stepanenko) [reply NEW_COMPONENT.md]: oops, thanks

**Comment by droshev** [comment]
@g-cheishvili I think it would be helpful for you to take a look this PR

**Review by InnaAtanasova — APPROVED** [review]

**Comment by g-cheishvili** [comment]
I have noticed one thing. MD files are not resolved when served if you do not run `copy-docs` script initially. Also after every update on those files they need to be re-run, otherwise you will see old copy of those files when serving. Also `cp` does not exist in windows environment and in the end scripts became depended on platform they are run. To fix this angular has it's [own way of dealing with such situations](https://angular.io/guide/workspace-config#assets-configuration). I think we should update angular.json to have symblinks and then those files will be automatically copied during build both on serve and build

**Review by g-cheishvili — APPROVED** [review]
Everything is easy to understand for a regular dev. No duplications and unified way of dealing with new components. Nicely done. I do not thing anything will pop up if merged

**Comment by g-cheishvili** [comment]
> I have noticed one thing. MD files are not resolved when served if you do not run `copy-docs` script initially. Also after every update on those files they need to be re-run, otherwise you will see old copy of those files when serving. Also `cp` does not exist in windows environment and in the end scripts became depended on platform they are run. To fix this angular has it's [own way of dealing with such situations](https://angular.io/guide/workspace-config#assets-configuration). I think we should update angular.json to have symblinks and then those files will be automatically copied during build both on serve and build

I think this should be done in separate issue, because some scripts will need modifications as well and this change will require additional testing. @droshev @InnaAtanasova @dmitry-stepanenko

**Comment by dmitry-stepanenko** [comment]
> > I have noticed one thing. MD files are not resolved when served if you do not run `copy-docs` script initially. Also after every update on those files they need to be re-run, otherwise you will see old copy of those files when serving. Also `cp` does not exist in windows environment and in the end scripts became depended on platform they are run. To fix this angular has it's [own way of dealing with such situations](https://angular.io/guide/workspace-config#assets-configuration). I think we should update angular.json to have symblinks and then those files will be automatically copied during build both on serve and build
> 
> I think this should be done in separate issue, because some scripts will need modifications as well and this change will require additional testing. @droshev @InnaAtanasova @dmitry-stepanenko

Thanks for the review, @g-cheishvili . I totally agree with everything you've mentioned above. Worth saying it occurs only in local development, as those scripts are included in CI. These problems were not presented by my changes and are not in scope of current PR, so yeah, we definitely need another one. Created https://github.com/SAP/fundamental-ngx/issues/7523

**Review by Betrozov — APPROVED** [review]

**Pull request merged**